package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "beesoft-common",
```

Comment:
Can we change this to `@beesoft/common`? I own the name on npm and can release scoped libraries. BTW if you want to see how to release a scoped library you can look at the beesoft-react-dragula repo.